recipe/meta.yaml

```diff
@@ -12,32 +11,18 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  number: {{ build }}
+  number: 0
   entry_points:
     - ipcluster = ipyparallel.apps.ipclusterapp:launch_new_instance
     - ipcontroller = ipyparallel.apps.ipcontrollerapp:launch_new_instance
     - ipengine = ipyparallel.apps.ipengineapp:launch_new_instance
+  script: pip install --no-deps .
```

Comment:
Should we add `--ignore-installed` as well?

ref: https://conda-forge.org/docs/meta.html#use-pip

Reply:
Can do. It doesn't do anything extra since the package isn't installed, but added for consistency with the docs.